CONTRIBUTING.md

```diff
@@ -64,34 +81,15 @@ Please fill out either the individual or corporate Contributor License Agreement
-just getting started, Github has a
-[how to](https://help.github.com/articles/using-pull-requests/).
-
-TensorFlow team members will be assigned to review your pull requests. Once the
-pull requests are approved and pass continuous integration checks, a TensorFlow
-team member will apply `ready to pull` label to your change. This means we are
-working on getting your pull request submitted to our internal repository. After
-the change has been submitted internally, your pull request will be merged
-automatically on GitHub.
-
-If you want to contribute, start working through the TensorFlow codebase,
-navigate to the
-[Github "issues" tab](https://github.com/tensorflow/tensorflow/issues) and start
-looking through interesting issues. If you are not sure of where to start, then
-start by trying one of the smaller/easier issues here i.e.
-[issues with the "good first issue" label](https://github.com/tensorflow/tensorflow/labels/good%20first%20issue)
-and then take a look at the
-[issues with the "contributions welcome" label](https://github.com/tensorflow/tensorflow/labels/stat%3Acontributions%20welcome).
-These are issues that we believe are particularly well suited for outside
-contributions, often because we probably won't get to them right now. If you
-decide to start on an issue, leave a comment so that other people know that
-you're working on it. If you want to help out, but not alone, use the issue
-comment thread to coordinate.
+If you have improvements to TensorFlow, send us your pull requests! For those just getting started, Github has a [how to](https://help.github.com/articles/using-pull-requests/).
+
+TensorFlow team members will be assigned to review your pull requests. Once the pull requests are approved and pass continuous integration checks, a TensorFlow team member will apply `ready to pull` label to your change. This means we are working on getting your pull request submitted to our internal repository. After the change has been submitted internally, your pull request will be merged automatically on GitHub.
+
+If you want to contribute, start working through the TensorFlow codebase, navigate to the Github ["issues"](https://github.com/tensorflow/tensorflow/issues) tab and start looking through interesting issues. If you are not sure of where to start, then start by trying one of the smaller/easier issues here i.e. issues with the ["good first issue"](https://github.com/tensorflow/tensorflow/labels/good%20first%20issue) label and then take a look at the issues with the ["contributions welcome"](https://github.com/tensorflow/tensorflow/labels/stat%3Acontributions%20welcome) label. These are issues that we believe are particularly well suited for outside contributions, often because we probably won't get to them right now. If you decide to start working on an issue, leave a comment so that other people know that you're working on it. If you want to help out, but not alone, use the issue comment thread to coordinate.
```

Comment:
Removing line breaks here makes it harder to review what changed. Please revert.

Reply:
Alright. In some cases, line breaks were needed because of the use of \- in the previous version. But I will make the changes as you suggested.

Reply:
We have automation for formatting and the changes would be reverted by that.

Reply:
So I will keep the line breaks like before. 👍🏻  Any suggestion about the mermaid diagram?